src/player.rs

```diff
@@ -165,13 +227,27 @@ impl Player {
                     .expect("Can't set video sink property");
             }
         }
+        */
+        let video_sink = gst::ElementFactory::make("appsink", None).unwrap();
+        let pipeline = player.get_pipeline().unwrap();
+        pipeline.set_property("video-sink", &video_sink).unwrap();
+        let video_sink = video_sink.dynamic_cast::<gst_app::AppSink>().unwrap();
+        video_sink.set_caps(&gst::Caps::new_simple(
+            "video/x-raw",
+            &[
+                ("format", &"BGRA"),
+                ("pixel-aspect-ratio", &gst::Fraction::from((1, 1))),
```

Comment:
Out of curiosity, why a 1/1 PAR?

Reply:
I didn't investigate yet where exactly I would tell servo to stretch the frames for PAR-correction. I *believe* this can be done by giving the pixel dimensions when creating the image, and the visible dimensions (after PAR-correction) in the display list.